extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -225,6 +225,11 @@ export class AmpGeo extends AMP.BaseElement {
             return GROUP_PREFIX + group;
           });
 
+
+          if (!self.matchedGroups_.length) {
```

Comment:
Nit: Since you're using fat arrow functions, the context is bound automatically to the ``amp-geo`` instance. I believe you could replace ``self`` by ``this`` in this ``waitForBodyPromise`` callback and it'd still work.

Reply:
It's a size thing, closure compiler will alias `self` to a single letter and given how many times it's used it's a net saving over `this`

Reply:
Thanks for the context :)